presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileFunctions.java

```diff
@@ -80,6 +82,32 @@ public void testSerialization()
         assertEquals(tile, objectMapper.readerFor(BingTile.class).readValue(json));
     }
 
+    @Test
+    public void testBingTileBigint()
+    {
+        Random random = new Random(1812);
```

Comment:
I would do this in a loop of say 1000 just to make sure we get good signal.

Reply:
Since this is testing a query, the setup/teardown meant that doing 1000 (particularly for each level) was really slow (a couple minutes on my laptop).  Is there a way to reduce the overhead, or is that kind of test length worth the coverage?

Reply:
Instead of 1000, any nontrivial positive number would probably work